script/Arcueid/Day 5/05_00_ARC05_9_1.txt

```diff
@@ -347,12 +359,12 @@ It is impossible for a human to cross those two steps.
 [sha:85c7330170dccd9a5ed9a4c8d9072a4e0b5fe704]{
 -- Page 797, Offset 10819.
 -- 悲鳴をあげるだけの余裕も、
-My ability to scream,
+My ability to scream―――
 }
 [sha:1a83fcc05740cf8545d9966a645e2c3ef3ed2347]{
 -- Page 797, Offset 10820.
 -- 苦痛を感じるだけの感覚も、既に凍っていた。
-and my capacity to feel pain were already frozen solid.
+and my capacity to feel pain are already frozen solid.
```

Comment:
caps

Reply:
no need for caps here, its the same sentence; it's just split

Reply:
We should probably actually agree on the rules here. I have been operating under "every new line starts with a capital letter"

Reply:
I don't really care all that much but it's not really a new sentence or an independent clause so I don't see a reason to capitalize it